modules/ROOT/pages/runtime-configure-gov.adoc

```diff
@@ -0,0 +1,83 @@
+= To Configure the Mule Runtime to Connect to the Government Cloud
```

Comment:
The instructions here are to enable API Gateway capabilities. You may use the Mule Runtime in other ways in the Government Cloud without specifying any property.

Reply:
Thank you - I've reworded the title.